File: Java/Guias/guia3/ej8_CellPhone/Call.java
package Guias.guia3.ej8_CellPhone;

public class Call {
    private String from, to;
    private int duration;

    private static final double costPerSecond = 0.01;

    Call(String from, String to, int duration) {
        this.from = from;
        this.to = to;
        this.duration = duration;
    }

    public double getCost() {
        return duration * costPerSecond;
    }

    public String getTo(){
        return to;
    }

}

File: RUBY/Guias/Guia10/Ej4_FriendCellPhone/call.rb
# frozen_string_literal: true

class Call
  COST_PER_SECOND = 0.01

  def initialize(from, to, duration)
    @from = from
    @to = to
    @duration = duration
  end

  def cost
    @duration * COST_PER_SECOND
  end

  attr_reader :to
end
